api/types/swarm/runtime.go

```diff
@@ -0,0 +1,19 @@
+package swarm
+
+// RuntimeType is the type of runtime used for the TaskSpec
+type RuntimeType string
+
+// RuntimeURL is the proto type url
+type RuntimeURL string
+
+const (
+	// RuntimeContainer is the container based runtime
+	RuntimeContainer RuntimeType = "container"
+	// RuntimePlugin is the plugin based runtime
+	RuntimePlugin RuntimeType = "plugin"
+
+	// RuntimeURLContainer is the proto url for the container type
+	RuntimeURLContainer RuntimeURL = "types.docker.com/RuntimeContainer"
```

Comment:
It looks like it's supposed to be a Url (https://github.com/gogo/protobuf/blob/master/types/any.pb.go#L109 -- example https://github.com/gogo/protobuf/blob/master/types/any.pb.go#L104).